dm/syncer/err-operator/operator.go

```diff
@@ -136,7 +198,11 @@ func (h *Holder) MatchAndApply(startLocation, endLocation binlog.Location, realE
 		return false, pb.ErrorOp_InvalidErrorOp
 	}
 
-	if operator.op == pb.ErrorOp_Replace {
+	if operator.isAllInjected {
```

Comment:
`MatchAndApply` is only called for DDL (query event), but I think we want to inject at any event? @sunzhaoyang @lichunzhu @GMHDBJD

Reply:
Maybe at `RowsEvent` first, and inject query just support ddl.